service/src/test/java/uk/gov/hmcts/reform/fpl/service/orders/generator/ManageOrdersClosedCaseFieldGeneratorTest.java

```diff
@@ -120,16 +125,50 @@ private CaseData buildCaseData(String closeCase, String isFinalOrder, Order orde
     @ParameterizedTest
     @MethodSource("generateOrderStates")
     void shouldCloseCaseWhenOrderIsDefaultFinal(String isFinalOrder) {
-        CaseData caseData = buildCaseData("Yes", isFinalOrder, C32A_CARE_ORDER);
+        CaseData caseData = buildCaseData("Yes", isFinalOrder, C32A_CARE_ORDER, APPROVAL_DATE, null);
 
```

Comment:
Just minor thing: can we add the test when both approval_date and approval_date_time are not null?

Reply:
Cool. Added a test. Could you approve?